packages/protocol/contracts/common/Accounts.sol

```diff
@@ -424,9 +643,14 @@ contract Accounts is
    * @return The address with which the account can sign attestations.
    */
   function getAttestationSigner(address account) public view returns (address) {
-    require(isAccount(account), "Unknown account");
-    address signer = accounts[account].signers.attestation;
-    return signer == address(0) ? account : signer;
+    return getDefaultSigner(account, AttestationSigner);
+  }
+
+  function hasDefaultSigner(address account, string memory role) public view returns (bool) {
+    require(isAccount(account));
```

Comment:
ditto

Reply:
`eth_call` reverting isn't super useful imo

Reply:
What if the view is used on a non-view function tho?